ament_cmake_export_libraries/cmake/environment/library_path.bat.in

```diff
@@ -31,4 +29,3 @@ goto:eof
     set "%~1=%_list%"
   )
 goto:eof
-
```

Comment:
The Linux equivalents don't print `Inside` / `Leaving` either. Instead whenever another script is called that location also calls `echo` before.